src/Symfony/Component/Validator/Constraints/MacAddressValidator.php

```diff
@@ -23,6 +23,37 @@
  */
 class MacAddressValidator extends ConstraintValidator
 {
+    /**
+     * Checks whether an MAC address is valid.
+     *
+     * @internal
+     */
+    public static function checkMac(string $mac, mixed $version): bool
```

Comment:
For `Cidr` & `Ip` constraint it was useful to allow such methods be accessed elsewhere. Thought this will not hurt, may someone needs such methods in own contraints (also if marked as internal)...

I can change it to private or back to validator-method, if preferred :-)

Reply:
should be moved back as private at least.

Marking it as `@internal` still forbids others to access it (as we could remove it at any time). And adding something "just in case" is a no-go as it then forces us to maintain BC for it during years.

Reply:
Thx, changed to private. What about `isUnicast()` and `isLocal()`? Should I also change them to private? For me it's fine to set them as public-internal, so when someone rly wants to use them to not have "dublicate code", he/she/it can use the methods in own risk. And these methods are rly generic and normally makes no sense to remove them without removing the whole "feature" from constraint. U decide :-)

Reply:
yes, they should be private. It does not make sense to have those as a public API in the constraint validator class.

We don't have a concept of "public-internal code that you can still use if you want". Internal code is not covered by the BC promise at all so we can refactor it in any way we want without notice (and this is an important to allow us to keep our sanity while maintaining the project).

Reply:
I see, changed & thx for fast reply! :-)